ui/app/src/Data/Alert.elm

```diff
@@ -0,0 +1,63 @@
+{-
+   Alertmanager API
+   API of the Prometheus Alertmanager (https://github.com/prometheus/alertmanager)
+
+   OpenAPI spec version: 0.0.1
+
+   NOTE: This file is auto generated by the openapi-generator.
+   https://github.com/openapitools/openapi-generator.git
+   Do not edit this file manually.
+-}
+
+
+module Data.Alert exposing (Alert, alertDecoder, alertEncoder)
+
+import Data.AlertStatus exposing (AlertStatus, alertStatusDecoder, alertStatusEncoder)
+import Data.Receiver exposing (Receiver, receiverDecoder, receiverEncoder)
+import DateTime exposing (DateTime, dateTimeDecoder, dateTimeEncoder)
+import Json.Decode as Decode exposing (Decoder)
+import Json.Decode.Pipeline exposing (optional, required)
+import Json.Encode as Encode
+import Maybe exposing (map, withDefault)
+
+
+type alias Alert =
```

Comment:
It would be great to update the spec to reflect required values so we can directly interact with the values we know the API will always return

Reply:
:+1: Adjusted accordingly across the entire spec.